stargz/stargz.go

```diff
@@ -171,6 +171,9 @@ type TOCEntry struct {
 	// Xattrs are the extended attribute for the entry.
 	Xattrs map[string][]byte `json:"xattrs,omitempty"`
 
+	// Digest stores the OCI checksum for regular files payload.
+	Digest string `json:"checksum,omitempty"`
```

Comment:
The field name "Digest" doesn't match the JSON name "checksum".

But I think I'd like to go back to naming it specifically what it is, for when the algorithm inevitably changes in a future version of the spec.

How about `FileSHA256` for field and `fileSHA256` for the JSON?

Reply:
I don't have a very strong opinion on this, but OCI has a pretty well-defined and open-ended [description of digests](https://github.com/opencontainers/image-spec/blob/master/descriptor.md#digests) that I think would introduce minimal churn. In the short term, I suspect everything will be "sha256:...", but it leaves room to move to a different algorithm in the future without having to rename this field.

Reply:
Oh, I see. I didn't know they did that. (My project Perkeep does the same)

Okay, then I'm fine with just updating the JSON field name and clarifying the docs, that it's of form "sha256:abcdef01234...."